merlin/dataloader/tensorflow.py

```diff
@@ -116,6 +116,9 @@ def __init__(
         drop_last=False,
         transforms=None,
         device=None,
+        use_row_lengths=False,
+        tensors_as_1d = True,
+        lists_as_tuple = False
```

Comment:
Not sure how others feel about this, but in the interest of simplifying the code I wonder if we can live without the extra configuration flags and just make the new behavior the default/standard

Reply:
Unless we're making the default values preserve the existing behaviour, it's going to be a breaking change that forces some action. I agree in this case it seems like a situation where not having the flags seems like a reasonable option in this case.